public/javascripts/SVValidate/src/status/StatusPopupDescriptions.js

```diff
@@ -0,0 +1,122 @@
+            case "example-image-4":
+                return "This fire hydrant blocks the sidewalk";
+            case "counterexample-image-1":
+                return "There is enough space to comfortably maneuver around this pole";
+            case "counterexample-image-2":
+                return "This pole located between, but not on these curb ramps";
+            case "counterexample-image-3":
+                return "This tree is not on the pedestrian path";
+            case "counterexample-image-4":
+                return "There is enough space to comfortably maneuver around this lid";
+        }
+    }
+
+    function getSurfaceProblemDescription (id) {
+        switch (id) {
+            case "example-image-1":
+                return "This sidewalk is very uneven";
+            case "example-image-2":
+                return "This sidewalk is very uneven";
+            case "example-image-3":
+                return "This sidewalk is cracked";
+            case "example-image-4":
+                return "There is a substantial amount of grass growing between the sidewalk cracks";
+            case "counterexample-image-1":
+                return "Surface problems on curb ramps should not be labeled as surface problems";
+            case "counterexample-image-2":
+                return "This gravel is not on the pedestrian path";
+            case "counterexample-image-3":
+                return "This grass is not on the pedestrian path";
+            case "counterexample-image-4":
+                return "Normal sidewalk tiles are not surface problems";
+        }
+    }
+
+    function getNoSidewalkDescription (id) {
+        switch (id) {
+            case "example-image-1":
+                return "This is an abruptly ending sidewalk";
+            case "example-image-2":
+                return "This is an abruptly ending sidewalk";
+            case "example-image-3":
+                return "This street is missing a sidewalk";
+            case "example-image-4":
+                return "This intersection is missing a sidewalk";
+            case "counterexample-image-1":
+                return "Narrow sidewalks should not be labeled as missing sidewalks";
+            case "counterexample-image-2":
+                return "Driveways should not be labeled as missing sidewalks";
+            case "counterexample-image-3":
+                return "Traffic medians should not be labeled as missing sidewalks";
+            case "counterexample-image-4":
+                return "Alleyways and side streets should not be labeled as missing sidewalks";
+        }
+    }
+
+    self.getCurbRampDescription = getCurbRampDescription;
+    self.getMissingCurbRampDescription = getMissingCurbRampDescription;
+    self.getObstacleDescription = getObstacleDescription;
+    self.getSurfaceProblemDescription = getSurfaceProblemDescription;
+    self.getNoSidewalkDescription = getNoSidewalkDescription;
+
+    return this;
+}
```

Comment:
aaand just missing a newline at the end of this file :wink:

Reply:
whoops! This is fixed now